Modules/Devices/test/Devices.Application.Tests/Tests/Identities/Queries/ListIdentities/HandlerTests.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Backbone.Modules.Devices.Infrastructure.Persistence.Database;
+using Enmeshed.BuildingBlocks.Application.Abstractions.Infrastructure.UserContext;
+using Enmeshed.DevelopmentKit.Identity.ValueObjects;
+using FakeItEasy;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using Xunit;
+using Backbone.Modules.Devices.Application.Tests;
+using Enmeshed.UnitTestTools.TestDoubles.Fakes;
+using Backbone.Modules.Devices.Domain.Entities;
+using Backbone.Modules.Devices.Application.Identities.Queries.ListIdentities;
+using Handler = Backbone.Modules.Devices.Application.Identities.Queries.ListIdentities.Handler;
+using Enmeshed.BuildingBlocks.Application.Pagination;
+using Backbone.Modules.Devices.Application.AutoMapper;
+
+namespace Backbone.Modules.Devices.Application.Tests.Tests.Identities.Queries.ListIdentities;
+
+public class HandlerTests
+{
+    private static readonly IdentityAddress ActiveIdentity = TestDataGenerator.CreateRandomIdentityAddress();
+
+    private readonly DevicesDbContext _arrangeContext;
+    private readonly DevicesDbContext _actContext;
+    private readonly Handler _handler;
+
+    public HandlerTests()
+    {
+        AssertionScope.Current.FormattingOptions.MaxLines = 1000;
+
+        (_arrangeContext, _, _actContext) = FakeDbContextFactory.CreateDbContexts<DevicesDbContext>();
+        _handler = CreateHandler();
+    }
+
+    [Fact]
+    public async void Returns_empty_list_when_no_loaded_entities()
```

Comment:
What do you mean with "loaded entities"? Wouldn't it be better like `Returns_an_empty_list_when_no_identities_exist`?